scripts/seed.py

```diff
@@ -134,6 +136,7 @@ def create_challenge(title, start_date, end_date, host_team):
     """
     evaluation_script = open(
         os.path.join(settings.BASE_DIR, 'examples', 'example1', 'sample_evaluation_script.zip'), 'rb')
+    random_string = ''.join(random.choice(string.ascii_letters) for _ in xrange(75))
```

Comment:
Also, there is no `xrange` in Python3. Please change it to `range`.

Reply:
About that, at the start of the file, there’s xrange = range for python so this should work ever for python 3.

Reply:
I see. But it would be nice if we follow only Python 3 syntax here. Can you please change it so that I can merge it.